### ISSUE TITLE
Your login session has expired

### ISSUE THREAD

**Issue by Blacksmoke16** [ccpgames/sso-issues#63]
Reference: https://github.com/Blacksmoke16/GESI/issues/63

It seems when trying to authenticate via Google Scripts, some users are running into an error when trying to login.

> Login failed. Possible reasons can be: Your login session has expired. Please try again.

Could be related to https://github.com/ccpgames/sso-issues/issues/20 or https://github.com/esi/esi-issues/issues/519#issuecomment-323892188.

**Comment by akazakou** [comment]
@Blacksmoke16 did you try to use refresh token? AFAIK: token will expire in ~15min and after it, we need to [use a refresh token](https://docs.esi.evetech.net/docs/sso/refreshing_access_tokens.html) to generate a new token.

**Comment by Blacksmoke16** [comment]
@akazakou Yes of course.  This issue happens when trying to go thru the SSO flow.  I.e. before you even get a token. E.g.

![image](https://user-images.githubusercontent.com/12136995/108238404-ee875d80-7116-11eb-8708-3e09eb721c7b.png)
